hack/make-rules/test.sh

```diff
@@ -268,16 +271,16 @@ produceJUnitXMLReport() {
 
   local test_stdout_filenames
   local junit_xml_filename
-  test_stdout_filenames=$(ls ${junit_filename_prefix}*.stdout)
+  test_stdout_filenames=$(ls "${junit_filename_prefix}"*.stdout)
   junit_xml_filename="${junit_filename_prefix}.xml"
   if ! command -v go-junit-report >/dev/null 2>&1; then
     kube::log::error "go-junit-report not found; please install with " \
       "go get -u github.com/jstemmer/go-junit-report"
     return
   fi
-  cat ${test_stdout_filenames} | go-junit-report > "${junit_xml_filename}"
+  go-junit-report < "${test_stdout_filenames}" > "${junit_xml_filename}"
```

Comment:
fix this failure
```
In ./hack/make-rules/test.sh line 278:
  cat ${test_stdout_filenames} | go-junit-report > "${junit_xml_filename}"
      ^----------------------^ SC2086: Double quote to prevent globbing and word splitting.
      ^----------------------^ SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
```